forge-gui/res/cardsfolder/upcoming/halo_fountain.txt

```diff
@@ -1,7 +1,7 @@
 Name:Halo Fountain
 ManaCost:2 W
 Types:Artifact
-A:AB$ Token | Cost$ W T untapYType<1/Creature.YouCtrl/creature> | TokenAmount$ 1 | TokenScript$ gw_1_1_citizen | TokenOwner$ You | LegacyImage$ gw 1 1 citizen snc | SpellDescription$ Create a 1/1 green and white Citizen creature token.
+A:AB$ Token | Cost$ W T untapYType<1/Creature.YouCtrl/creature> | TokenScript$ gw_1_1_citizen | SpellDescription$ Create a 1/1 green and white Citizen creature token.
 A:AB$ Draw | Cost$ W W T untapYType<2/Creature.YouCtrl/creatures> | SpellDescription$ Draw a card.
 A:AB$ WinsGame | Cost$ W W W W W T untapYType<15/Creature.YouCtrl/creatures> | Defined$ You | SpellDescription$ You win the game.
 DeckHints:Type$Citizen
```

Comment:
For this one, I think keep this line, but then in the next line also add `& Type$Citizen`